Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest/ContestDetailsPage.tsx

```diff
@@ -125,7 +149,7 @@ const ContestDetailsPage = () => {
     const renderContestButtons = useCallback(
         () => (
             <div className={styles.buttonsContainer}>
-                {contestDetails?.canViewResults || contestDetails?.isAdminOrLecturerInContest
+                {(contestDetails?.canViewResults || contestDetails?.isAdminOrLecturerInContest) && !isNil(participationType)
```

Comment:
Participation type should not determine anything in the contest details page. In the legacy system there are two separate buttons for results.
![image](https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/19411274/48e91622-f784-429b-b9ef-04aff97bbe9d)

Reply:
Replaced with ContestParticipationType.Compete.